Automattic/newspack-block-theme
# feat: add backward compatibility with classic theme

### All Submissions:






### Changes proposed in this Pull Request:

This PR matches the old variables (primary, secondary, etc...) with the new ones (accent, accent-2, etc...). It also makes sure normal and huge font-sizes are rendered correctly.

Note: In the editor the colour swatch or font-size will return empty. This is normal/expected.

### How to test the changes in this Pull Request:

1. Using the Classic Theme, create a page with a bunch of groups, columns, etc... with colours, background colours, borders
2. Also play with font-sizes
3. Switch to the Block Theme (trunk)
4. You should not see your primary colour etc...
5. Switch to this branch and refresh page

### Other information:
